x-pack/plugins/security_solution/public/resolver/models/process_event.ts

```diff
@@ -88,3 +99,62 @@ export function uniqueParentPidForProcess(passedEvent: ResolverEvent): string |
     return passedEvent.process.parent?.entity_id;
   }
 }
+
+/**
+ * Returns the process event's parent pid
+ */
+export function processParentPid(passedEvent: ResolverEvent): string | undefined {
+  if (event.isLegacyEvent(passedEvent)) {
+    return String(passedEvent.endgame.unique_ppid);
+  } else {
+    const ppid = passedEvent.process.parent?.pid;
```

Comment:
They are not, since this is just displaying the info and not concerned with anything that could run into issues with pid reuse, just use the normal ppid value https://github.com/elastic/kibana/blob/master/x-pack/plugins/security_solution/common/endpoint/types.ts#L396

Reply:
Thanks h/t @kqualters-elastic 2445aba